README.md

```diff
@@ -1,43 +1,31 @@
 # Rancher Logging Examples
 
-This repository contains 2 example charts for testing your logging setup and a simple application for generating logs.
+This repository contains an example charts for testing your logging setup.
 
 ## Charts
 
-### Simple Application
-The `simple-application` chart will deploy 2 pods that will constantly out logs.  This chart is recommended for when you want to configure your own logging setup and need a simple application to collect logs from.
-
 ### Rancher Logging Example
-The `rancher-logging-example` is an end to end setup of an application, multiple configurations for log collection, and an elasticsearch cluster as an output.  This chart is recommended if you want to see a working example of the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart.  
+The `rancher-logging-example` is an end to end setup of an application, multiple configurations for log collection, and a simple output.  This chart is recommended if you want to see a working example of the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart.  
 
 This chart demonstrates 3 scenarios:
 
 1. `all-logs` collects all logs in the cluster
 2. `logs-by-label` collects all logs for pods with the label `rancher.logging.example/app: log-generator`
 3. `namespace-logs` collects all logs for the namespace this chart is deployed in.
 
-#### Resource Requests and Limits
-
-You can change `elasticsearch` resource requests and limits from the upstream chart by adding the following `helm install` arguments...
+#### Installation
+To install this chart run:
 
-```bash
-helm install <required-and-optional-args> \
-    --set elasticsearch.resources.requests.cpu="0" \
-    --set elasticsearch.resources.requests.memory="0" \
-    --set elasticsearch.resources.limits.cpu="0" \
-    --set elasticsearch.resources.limits.memory="0"
+```shell
+git clone https://github.com/paynejacob/rancher-logging-examples
+cd rancher-logging-examples
+make install

```

Comment:
nit: Don't know how much this matters, but I usually default to saying "clone the repository, `cd` into it, and run..." since you can clone via SSH or HTTPS.

Reply:
I prefer to post with https because anyone can do it on any machine.  This way users that don't want to read and want to get something running as quickly as possible can find a block to copy and paste.